eslint.config.js

```
@@ -0,0 +1,118 @@
+import js from '@eslint/js';
+import reactPlugin from 'eslint-plugin-react';
+import reactHooksPlugin from 'eslint-plugin-react-hooks';
+import importPlugin from 'eslint-plugin-import';
```

ESLint

[error] `eslint-plugin-import` import should occur before import of `eslint-plugin-react`

(import/order)